pkg/db/id.go

```diff
@@ -17,9 +17,9 @@ type IDWriter interface {
 	SetID(ID) error
 }
 
-func NewID(age time.Time) ID {
+func NewID(age time.Time, schemaVersion int) ID {
 	return ID{
 		BuildTimestamp: age.UTC(),
-		SchemaVersion:  SchemaVersion,
+		SchemaVersion:  schemaVersion,
```

Comment:
This is derived from a constant, and that constant is tied to the codebase directly (since the codebase only works for the schema it is coded to). With this change it would now be possible to supply a schema version that this code does not support, yes?